lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Folders.pm

```diff
@@ -26,21 +41,25 @@ sub index {
         $folders
           = Mojo::File->new($helper->home, $obs_project // '')->list({dir => 1})->grep(sub { -d $_ })->map('basename')
           ->to_array;
-        @$folders = grep { !/^(t|profiles|script|xml)$/ } @$folders;
+        @$folders = grep { !/$non_project_folders/ } @$folders;
     }
 
     for my $folder (@$folders) {
         my $alias = $folder;
         $alias = "$obs_project|$folder" if $obs_project;
         my $run_last_info = $helper->get_run_last_info($alias);
         my ($fail_last_job_id, $fail_last_when) = $helper->get_fail_last_info($alias);
+        my $dirty_status = $helper->get_dirty_status($obs_project // $folder);
+        my $api_repo     = $helper->get_api_repo($obs_project     // $folder);
+        $dirty_status = "$dirty_status ($api_repo)" if $api_repo ne "images";
```

Comment:
Why is `images` excempt here? :thinking:

Reply:
`images` is default value, because it is used in most of projects.